rpc/tests/trace/sepoliaSimulateInvokeTxResp.json

```diff
@@ -182,7 +182,7 @@
 					"range_check_builtin_applications": 121,
 					"ec_op_builtin_applications": 3,
 					"data_availability": {
-						"l1_gas": 4668,
+						"l1_gas": 4646,
```

Comment:
looks like it should indeed be 4668: https://github.com/NethermindEth/starknet.go/actions/runs/11381954197/job/31664465265?pr=639#step:8:68

Reply:
oh I just saw Rian's comment now.

In this case we need to update the endpoint that starknet.go is using

Reply:
The endpoint under test is using a buggy version of Juno that doesn’t yet include the [fix](https://github.com/NethermindEth/juno/pull/2118).

Reply:
We decided to hold off on merging it until the endpoint is updated with the new Juno version.